source/rmarkdown/annelida_data_analyse.Rmd

```diff
@@ -430,6 +528,406 @@ conditional_effects(
   re.form = NA)
 ```
 
+### Staalname maand als verklarende variabele
+
+test
+
+```{r m-annelida-richness-maand}
+samples_totcount_10k <- tidy_physeq$samples %>%
+  filter(total_count > 1e+4)
+
+form_annelida_richness <- formula(
+  observed ~
+    log(total_count)
+  + Maand_staalname
+  + Diepte
+  + Maand_staalname:Diepte
+  + (1 | Cmon_PlotID)
+  )
+
+m_annelida_richness <- glmmTMB::glmmTMB(
+  formula = form_annelida_richness,
+  data = samples_totcount_10k,
+  family = poisson())
+```
+
+Model validatie OK.
+
+
+```{r}
+performance::check_model(m_annelida_richness)
+```
+
+
+```{r m-annelida-richness-summary-maand}
+summary(m_annelida_richness)
+```
+
+```{r m-annelida-richness-anova-maand}
+car::Anova(m_annelida_richness)
+```
+
+Zoals te verwachten, is het belangrijk om te corrigeren voor het totaal aantal reads:
+
+```{r}
+marginaleffects::plot_predictions(
+  m_annelida_richness,
+  condition = c("total_count"),
+  vcov = vcov(m_annelida_richness)) +
+  labs(y = "Voorspeld aantal taxa") +
+  scale_x_log10()
+```
+
+
+We kunnen nu predicties maken van soortenrijkdom waarbij we controleren voor het totaal aantal reads in een staal:
+
+```{r m-annelida-richness-predictions-maand}
+marginaleffects::plot_predictions(
+  m_annelida_richness,
+  condition = c("Maand_staalname", "Diepte"),
+  vcov = vcov(m_annelida_richness),
+  re.form = NA,
+  type = "response") +
+  labs(y = "Voorspelde aantal taxa")
+```
+
+### Staalname maand als covariaat
```

Comment:
De titel strookt niet met wat volgt

Reply:
Bedoel je dat staalname maand geen covariaat is als we de interactie met landgebruik verwijderen uit het model?

Reply:
Nee. Ik denk dat ik hiermee bedoelde  dat er naast staalname maand ook nog andere covariaten in het model zitten dat in deze sectie gefit werd. Terwijl de titel suggereert dat enkel staalname maand als covariaat bekeken wordt.